docs/pages/database-access/guides/elastic.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Database Access with Elasticsearch
+description: How to configure Teleport Database Access with Elasticsearch.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.3"
+>
+  Database access for Elasticsearch is available starting from Teleport `10.3`.
+</Details>
+
+This guide will help you to configure secured access to an Elasticsearch database using Teleport Dabatase Access.
+
+## Prerequisites
+
+- A self-hosted Elasticsearch database. Elastic cloud [does not support client certificates](https://www.elastic.co/guide/en/cloud/current/ec-restrictions.html#ec-restrictions-security).
+
+- A host where you will run the Teleport Database Service. Teleport version 10.3
+  or newer must be installed. <ScopedBlock scope={["oss", "enterprise"]}> This can also be the same instance of Teleport running your Auth and/or Proxy service(s).</ScopedBlock>
+
+  See [Installation](../../installation.mdx) for details.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service. Make sure you've selected your installation type (OSS, Enterprise, Clo):
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+<Tabs>
+<TabItem label="Standalone Binary">
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:3080 \
+  --name=myelastic \
+  --protocol=redis \
+  --uri={ELASTICSEARCH_HOST}:9200 \
```

Comment:
It may be unclear that `{ELASTICSEARCH_HOST}` is meant to be replaced with an actual string like `localhost` or `elasticsearch.example.com`. I wonder if using an example hostname would be better, i.e. `elasticsearch.example.com` or similar.

Reply:
replaced instances of {ELASTICSEARCH_HOST}` with `elasticsearch.example.com`. While I was at it, replaced similar password stand-in with `your_elasticsearch_password`